deepfence_bootstrapper/cgroups/cgroups.go

```diff
@@ -14,12 +14,12 @@ import (
 )
 
 var (
-	cgroups1            = map[string]cgroup1.Cgroup{}
-	cgroups2            = map[string]*cgroup2.Manager{}
-	FailUpdateError     = errors.New("Failed to update")
-	FailCreateError     = errors.New("Failed to create")
-	CgroupNotExistError = errors.New("Cgroup does not exist")
-	cgroupV2            bool
+	cgroups1          = map[string]cgroup1.Cgroup{}
+	cgroups2          = map[string]*cgroup2.Manager{}
+	ErrFailUpdate     = errors.New("failed to update")
+	ErrFailCreate     = errors.New("failed to create")
+	ErrCgroupNotExist = errors.New("cgroup does not exist")
```

Comment:
I would rather keep error message starting with caps, easier to spot when embedded in other messages

Reply:
on the contrary. Golang usually goes with chaining in logging akin to `err3: err2: err1` and actively encourages lowercase. On top of it if you include error messages within logs it may make sentences weird with sudden capitalization.